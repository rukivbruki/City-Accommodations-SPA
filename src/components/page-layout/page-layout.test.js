import React from "react";
import renderer from "react-test-renderer";
import PageLayout from "./page-layout";

const children = <div className="children-component" />;

it(`PageLayout component render correct`, () => {

  const layout = renderer
    .create(
        <PageLayout pageClasses={[`page--gray`, `page--main`]}>
          {children}
        </PageLayout>
    );

  expect(layout).toMatchSnapshot();
});
