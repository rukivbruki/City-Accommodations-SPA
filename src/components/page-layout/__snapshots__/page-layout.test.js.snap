// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLayout component render correct 1`] = `
<div
  className="page page--gray page--main"
>
  <div
    className="children-component"
  />
</div>
`;
